server/sonar-web/src/main/js/apps/projectActivity/components/forms/AddGraphMetricPopup.tsx

```diff
@@ -33,16 +34,41 @@ interface Props {
 }
 
 export default function AddGraphMetricPopup(props: Props) {
```

Comment:
It could make sens to extract some of the props for direct use (without "props."). I'd probably do : 
```ts
export default function AddGraphMetricPopup({ elements, metricsTypeFilter, ...props }: Props) {
```